src/formatDate.js

```diff
@@ -8,7 +8,61 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.find(
+    (item) => item === '/' || item === '-' || item === '.',
+  );
+
+  const separator = toFormat.find(
+    (item) => item === '/' || item === '-' || item === '.',
+  );
+
+  const newDate = date.split(fromSeparator);
+
+  const newArray = [];
+
+  const dateObject = {};
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    dateObject[fromFormat[i]] = newDate[i];
+  }
+
+  const indexOfYear = fromFormat.includes('YYYY')
```

Comment:
```suggestion
  const indexOfYear = fromFormat.includes(LONG_YEAR_FORMAT)
```
Consider using variables instead of magic strings